File: Backend/modules/boot-module/src/main/java/com/example/demo/api/MyConfiguration.java
package com.example.demo.api;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.example.demo.core.Sample;
import com.example.demo.core.SampleImpl;

@Configuration
public class MyConfiguration {
  @Bean
  public Sample sample() {
    return new SampleImpl();
  } 
}

File: Backend/modules/boot-module/src/main/kotlin/MyConfiguration.kt
package com.example.demo.api;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.example.demo.core.Sample;
import com.example.demo.core.SampleImpl;

@Configuration
class MyConfiguration {
  @Bean
  fun sample(): Sample {
    return SampleImpl();
  }
}
